compass/ocean/tests/sphere_transport/nondivergent_2d/forward.py

```diff
@@ -0,0 +1,100 @@
+import time
+
+from compass.model import run_model
+from compass.step import Step
+from datetime import timedelta
+
+
+class Forward(Step):
+    """
+    A step for performing forward MPAS-Ocean runs as part of the
+      nondivergent_2d test case.
+
+    Attributes
+    ----------
+    resolution : int
+        The resolution of the (uniform) mesh in km
+    """
+
+    def __init__(self, test_case, resolution, dt_minutes):
+        """
+        Create a new step
+
+        Parameters
+        ----------
+        test_case :
+            compass.ocean.tests.global_convergence.nondivergent_2d.Nondivergent2D
```

Comment:
Same comment about line length -- this needs to go back to being one line.
And again, the description got chopped in the last commit